tools/packaging/packaging.psm1

```diff
@@ -1173,7 +1173,7 @@ function Get-NugetSemanticVersion
     cd $RootPathOfPowerShellCheckout
     Import-Module .\build.psm1
     Start-PSBuild
-    Import-Module .\tools\packaging\packaging.psd1
+    Import-Module .\tools\packaging\packaging.psm1
```

Comment:
this line is no longer needed since it's part of packaging.psm1 already

Reply:
I see what you mean but the idea was to provide a copy-pasteable example that works out of the box. Would it be OK to rather squash the initialization as `Import-Module .\build.psm1; Start-PSBuild; Import-Module .\tools\packaging\packaging.psm1`? Otherwise I'll of course remove the line if you insist.

Reply:
For the comment based help, you would only see that if you already have the packaging module loaded, so it just seems weird.  I think if you want to provide an easy cut-paste, perhaps it's better to have it in https://github.com/PowerShell/PowerShell/blob/master/docs/building/windows-core.md